content/2009/eclipsecon/images/eclipse-foundation-grey-orange.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We can remove this file. This was added for the custom footer but it is now added globally so we don't need to add it to all the archived websites.

Reply:
Removed, thx!